components/MainDonationForm.js

```diff
@@ -115,6 +122,12 @@ export default function MainDonationForm() {
             options={AMOUNTS}
             value={amount}
           />
+          <ReCaptchaV2
+            id="captcha"
+            onChange={(token) => setReCaptchaToken(token)}
+            onExpired={() => setReCaptchaToken('')}
+            sitekey={'6LcjyaccAAAAAAjE2z6vBhs2tOdhC-t0F2OCMj7z'} //need to use env variables process.env.REACT_APP_SITE_KEY
```

Comment:
Here too!!

Reply:
I will delete it from the ReCaptcha website as we need to generate it again from the official account.